cmd/joy/release.go

```diff
@@ -222,9 +241,12 @@ func NewReleasePromoteCmd(preRunConfigs PreRunConfigs) *cobra.Command {
 	cmd.Flags().StringSliceVar(&templateVars, "template-var", nil, "Variable to pass to PR/commit templates, in the form of key=value (flag can be specified multiple times)")
 	cmd.Flags().BoolVarP(&narrow, "narrow", "n", false, "Use narrow columns mode")
 	cmd.Flags().BoolVarP(&wide, "wide", "w", false, "Use wide columns mode")
+	cmd.Flags().BoolVar(&all, "all", false, "select all releases")
+	cmd.Flags().BoolVar(&keepPreRelease, "keep-pre-release", false, "do not promote releases that are pre-releases in target env")
```

Comment:
With the usage of `--omit` should we call this `--omit-pre-release`?

```suggestion
	cmd.Flags().BoolVar(&keepPreRelease, "omit-pre-release", false, "Do not promote releases that are pre-releases in target env")
```

Reply:
omit is not about prereleases, its about omitting any release. So you could do:

```
joy rel prom --all --omit servicing
```

Reply:
I find the directionality very confusing.

Are we omitting prereleases from the source? Are omitting updates to the target if its a prerelease?

To me "keep" or "preserve" indicates to my brain that we are talking about the target environment... Although the english language is ambiguous.

Reply:
I'd be OK with that, sorry David I know it's my fault we went with "pre-release"! ;)

Reply:
> Are we omitting prereleases from the source? Are omitting updates to the target if its a prerelease?

You got a point, I would argue that is why there is a flag description. 

In practice, QA (the source in 99% of cases) has only regular releases. 🤷‍♂️

Reply:
I admit that "keep" has no ambiguity, whereas "omit" definitely has

Reply:
My thought it that both the `--keep-prerelease` and `--omit` both more or less do the same thing (preventing promotions), but have different names:

`... --all --keep-prerelease --omit accounts --omit salesforce`

That being said, if you guys are adamant about it, I won't block you.